lib/kubernetes-deploy/kubernetes_resource/service.rb

```diff
@@ -11,16 +11,19 @@ def sync
     end
 
     def status
-      if @num_pods_selected.blank?
+      if !requires_endpoints?
```

Comment:
Is it true that an endpoint is not required when the service is in type ExternalName or deployment replica is 0?
The if statement here is not quite straightforward, some comments will help to clarify.

Reply:
> Is it true that an endpoint is not required when the service is in type ExternalName or deployment replica is 0?

Yes, that's exactly it, and that already seems pretty clear to me by reading this method's name and definition. Here, making the status say "Doesn't require any endpoints" when `!requires_endpoints?` also seems straightforward. I wouldn't add any more comments.